src/professions/admin/check-your-answers.controller.ts

```diff
@@ -96,7 +97,7 @@ export class CheckYourAnswersController {
       legislations,
       confirmed: isConfirmed(draftProfession),
       captionText: ViewUtils.captionText(isConfirmed(draftProfession)),
-      isUK: isUK(version.occupationLocations),
+      isUK: isUK(version.occupationLocations || []),
```

Comment:
Again, could we not do this?

```suggestion
      isUK: version.occupationLocations ? isUK(version.occupationLocations) : false,
```

Or, better still, just return `false` in `isUK` if `occupationLocations` is undefined.

Reply:
Think I'm gonna err towards being a "good caller" and not passing in undefined